MetroscopeModelingLibrary/MultiFluid/HeatExchangers/FuelHeater.mo

```diff
@@ -126,6 +129,12 @@ equation
   assert(pinch > 0, "A negative pinch is reached", AssertionLevel.warning); // Ensure a positive pinch
   assert(pinch > 1 or pinch < 0,  "A very low pinch (<1) is reached", AssertionLevel.warning); // Ensure a sufficient pinch
 
+  // Nominal temperature differences
+  if nominal_DT_default then
+    nominal_cold_side_temperature_rise = hot_side.T_in - cold_side.T_in; // Corresponds to the maximum achievable temperature difference
+    nominal_hot_side_temperature_drop = hot_side.T_in - cold_side.T_in; // Corresponds to the maximum achievable temperature difference
```

Comment:
There is a copy paste issue, both in equation and in comment here 😇

Reply:
It is supposed to be the same 🤔